src/components/providers/authentication-provider.tsx

```diff
@@ -5,7 +5,7 @@ import Keycloak from "keycloak-js";
 import { useCallback, useEffect } from "react";
 
 interface Props {
-  children: JSX.Element;
+  children: JSX.Element[];
```

Comment:
Is this change necessary?

Reply:
The code started working after this change

Reply:
I removed it locally and it doesn't seem to cause any problems, also wasn't it working without this previously and it doesn't look like any other changes have being made in this file?

Reply:
Indeed does work without this change. Reversed the change.